Samples~/Cinemachine Example Scenes/Scenes/2D/Zoom2D.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using Cinemachine;
+using UnityEngine;
+
+public class Zoom2D : MonoBehaviour
+{
+    private CinemachineVirtualCamera vcam;
+    public float zoomSpeed;
+    void Start()
+    {
+        vcam = GetComponent<CinemachineVirtualCamera>();
+    }
+    private void Update()
+    {
+            float zoom = Input.mouseScrollDelta.y;
+            vcam.m_Lens.OrthographicSize = Mathf.Lerp(vcam.m_Lens.OrthographicSize, vcam.m_Lens.OrthographicSize + zoom * zoomSpeed, Time.deltaTime);
+        
```

Comment:
You need to clean up this method.  Add range checking.  I can create jitter by zooming to negative values.